src/main/java/alchemyplusplus/block/complex/distillery/DistilleryBlock.java

```diff
@@ -26,17 +26,14 @@ public DistilleryBlock(String blockname)
     public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer player, int meta, float b, float c, float g)
     {
         TileEntity tileEntity = world.getTileEntity(x, y, z);
-        if (tileEntity != null && !player.isSneaking())
```

Comment:
Also, where else do all of those `TileEntity`s get called? (specifically)
Another thing; how much would need to be replaced to fix all of these calls, and where? (related to prior question)